x/crosschain/keeper/msg_server_migrate_tss_funds.go

```diff
@@ -136,6 +136,7 @@ func (k Keeper) MigrateTSSFundsForChain(
 			TssNonce:               0,
 			GasLimit:               1_000_000,
 			GasPrice:               medianGasPrice.MulUint64(2).String(),
+			GasPriorityFee:         priorityFee.String(),
```

Comment:
We can have a multiplier for important transactions like this , so as to increase the chances of their success.

Reply:
I like this idea the multiplier could be dynamically calculated depending on the receiving chain usage.

Reply:
I'll add a hard-coded multiplier for now, opened an issue for a more complex logic. https://github.com/zeta-chain/node/issues/2528

Reply:
https://github.com/zeta-chain/node/pull/2483/commits/2ceb5b0b0290da52b3b6c0860afe828b4c58e3e0